CleverPushLocation/Source/CleverPushLocation.m

```diff
@@ -26,6 +35,12 @@ + (void)init {
                                 CLRegion *region = [[CLCircularRegion alloc]initWithCenter:center
                                                                                     radius:[[geoFence objectForKey:@"radius"] longValue]
                                                                                 identifier:[geoFence valueForKey:@"_id"]];
+                                double num = [[geoFence objectForKey:@"delay"]doubleValue];
```

Comment:
Code Style:
- Use a better variable name than `num`
- Space before ` doubleValue`

Logic:
- What is `delay` key is nil? I think we need to check `nil` before calling `doubleValue` otherwise might throw an exception

Reply:
Done